lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -733,12 +733,12 @@ func (h *StorageHandlersImpl) StatPath(params storage.StatPathParams) middleware
 
 }
 
-// ListImages returns a list of images in a store
+// GetImageStorageUsage returns images storage usage for this VCH
```

Comment:
Does calculation distinguish volume stores from differen VCH?

Reply:
What if I specify --volume-store sharedVmfs-0/<VCH name>:default for better folder struction?

Reply:
Good catch. Fixed.